helm/Chart.yaml

```diff
@@ -1,5 +1,7 @@
 apiVersion: "v1"
-name: opwen_cloudserver
+kind: Chart
+# Below name change fixes the error 'directory name and chart name must be the same'
+name: helm
```

Comment:
Renaming the chart may cause issues during our next deployment to the cluster. As such, I would recommend to avoid renaming the chart and instead create a new nested directory like this:

```
helm
└── opwen_cloudserver
    ├── Chart.yaml
    ├── templates
    │   ├── api-autoscaler.yaml
    │   ├── api-deployment.yaml
    │   ├── api-service.yaml
    │   ├── cluster-issuer.yaml
    │   ├── _helpers.tpl
    │   ├── ingress.yaml
    │   ├── nginx-deployment.yaml
    │   ├── nginx-service.yaml
    │   ├── webapp-deployment.yaml
    │   ├── webapp-service.yaml
    │   ├── worker-autoscaler.yaml
    │   └── worker-deployment.yaml
    └── values.yaml
```

We can then update each location that invokes the helm command in `upgrade.sh`, `setup.sh` and `makefile` to point to the new subfolder.

This lets us fix the lint issue without having to make any changes that may affect production.